frontend/src/pages/account-settings/account-settings.tsx

```diff
@@ -71,10 +71,24 @@ const AccountSettings: React.FC = () => {
           <div className={styles['container']}>
             <div className={styles['page-title']}>Account Settings</div>
             <div className={styles['page-nav']}>
-              <Link to={AppRoute.PROFILE} className={styles['profile-link']}>
+              <Link
+                to={AppRoute.PROFILE}
+                className={
+                  location.pathname === AppRoute.PROFILE
+                    ? styles['profile-link-active']
+                    : undefined
+                }
+              >
                 Profile
               </Link>
-              <Link to={AppRoute.LOGIN} className={styles['profile-link']}>
+              <Link
+                to={AppRoute.LOGIN}
+                className={
```

Comment:
same

Reply:
And maybe you may just use `:active` pseudo-class